src/braket/aws/aws_quantum_task.py

```diff
@@ -500,8 +505,20 @@ def _(
             paradigmParameters=paradigm_parameters
         )
 
+    serialization_properties = OpenQASMSerializationProperties(
+        qubit_reference_type=(
+            QubitReferenceType.PHYSICAL if disable_qubit_rewiring else QubitReferenceType.VIRTUAL
```

Comment:
I think we also need to check if the circuit is verbatim

Reply:
Yes, confirmed this.